resolwe_bio/processes/support_processors/gtf_to_bed.py

```diff
@@ -15,12 +15,15 @@
 
 
 class GTFtoBED(Process):
-    """GTF to BED conversion."""
+    """GTF to BED conversion.
+
+    Note that this process only works with ENSEMBL annotations.
```

Comment:
Add additional check that validates: `inputs.annotation.output.source == "ENSEMBL"`